jekyll/_cci2/credits.md

```diff
@@ -35,15 +35,70 @@ To set up your desired plan, go to `Settings > Plan Overview` in the CircleCI we
 ## Free plan
 {: #free-plan }
 
-As with the CircleCI legacy Container plan, CircleCI also supports a free-tier with the usage-based plan. You can still make use of many key features offered by CircleCI:
-
-- Using Orbs
-- Workspaces
-- Dependency Caching
-- Windows/Linux builds
+As with the CircleCI legacy Container plan, CircleCI also supports a free-tier with the usage-based plan. By using the Free Plan, you can take advantage of a large number of premium features that will allow your team to be more productive, efficient and fast.
```

Comment:
```suggestion
As with the CircleCI legacy Container plan, CircleCI also supports a free-tier with the usage-based plan. By using the Free plan, you can take advantage of a large number of premium features that will allow your team to be more productive, efficient and fast.
```
I believe we're capitalizing as follows, for consistency: "Free plan," "Performance plan," "Scale plan"